specification/security/resource-manager/Microsoft.Security/stable/2018-06-01/security.json

```diff
@@ -0,0 +1,275 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Security Center",
+        "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+        "version": "2018-06-01"
+    },
+    "host": "management.azure.com",
+    "schemes": ["https"],
+    "consumes": ["application/json"],
+    "produces": ["application/json"],
+    "security": [{
+        "azure_auth": [
+            "user_impersonation"
+        ]
+    }],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Security/pricings": {
+            "get": {
+                "x-ms-examples": {
+                    "Get pricings on subscription": {
+                        "$ref": "./examples/Pricings/ListPricings_example.json"
+                    }
+                },
+                "tags": ["Pricings"],

```

Comment:
should this support paging?

Reply:
Probably not.
It's limited to a closed list of pricing configurations. There are currently 3, expected to grow slightly but not by a lot.